packages/theme-chalk/src/color-picker-panel.scss

```
@@ -89,6 +89,62 @@
+    transform: translate(-50%, -50%);
+    width: 8px;
+    height: 8px;
+    border-radius: 50%;
+    border: 2px solid #fff;
+    box-shadow: 0 0 3px rgba(0, 0, 0, 0.2);
+    cursor: grab;
+    transition:
+      transform 0.2s,
+      box-shadow 0.2s;
+
+    &:hover {
```

Stylelint (17.13.0)

[error] Missing scoping root (nesting-selector-no-missing-scoping-root)

(nesting-selector-no-missing-scoping-root)